Patches/PsiTech/HediffDefs/PsiTechBuffs.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+	<Operation Class="PatchOperationFindMod">
+		<mods>
+			<li>PsiTech</li>
+		</mods>
+		<match Class="PatchOperationSequence">
+			<operations>
+				<!-- == PTMeleeMastery == -->
+				<!-- statOffsets -->
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/HediffDef[defName="PTMeleeMastery"]/stages/li[1]/statOffsets</xpath>
```

Comment:
If it's possible to target this `li` more specifically than by list position, it's preferable to do so, to avoid the potential for odd behavior.